WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -443,6 +457,28 @@ private extension ProductVariationsViewController {
     }
 }
 
+// MARK: Action Sheet
+//
+private extension ProductVariationsViewController {
+    @objc private func presentMoreOptionsActionSheet(_ sender: UIBarButtonItem) {
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .text
+
+        let editAttributesAction = UIAlertAction(title: Localization.editAttributesAction, style: .default) { [weak self] _ in
+            self?.navigateToEditAttributeViewController()
```

Comment:
Here, we call `navigateToEditAttributeViewController` which inside calls 
```swift
EditAttributesViewModel(product: product, allowVariationCreation: true)
```

For the edit flow, the `EditAttributesViewModel` should not allow the variation creation, as the way to create attributes is by tapping the `Add Variation` button once there is at least one variation.